examples/sample-input-minimal.tfvars

```diff
@@ -33,9 +33,9 @@ container_registry_admin_enabled    = false
 
 # AKS config
 kubernetes_version         = "1.23.8"
-default_nodepool_min_nodes = 1
+default_nodepool_min_nodes = 2
+default_nodepool_vm_type   = "Standard_D8s_v4"
```

Comment:
This value does not match the Jira story of this issues vm size. What was the need for the increase? Asking as it now affects the customer by have 2x the machines which translates to 8x the cost of the default node pool which typically has minimal impact on cost.

Reply:
@thpang Good catch, I entered the wrong value for vm_type on Friday, the value that I tested with is the value reflected in this PR Changes section and the Jira story, I've corrected it in the last commit.